src/Expo.php

```diff
@@ -38,6 +41,22 @@ public function __construct(DriverManager $manager = null)
         $this->client = new ExpoClient();
     }
 
+    /**
+     * Registers macro for handling each token with DeviceNotRegistered error separately
+     * @param $callback
+     */
+    public static function addDeviceNotRegisteredHandler($callback) {
+        self::macro('deviceNotRegistered', $callback);
+    }
+
+    /**
+     * Registers macro for handling all tokens with DeviceNotRegistered errors
+     * @param $callback
+     */
+    public static function addDevicesNotRegisteredHandler($callback) {
+        self::macro('devicesNotRegistered', $callback);
+    }
+
```

Comment:
One macro is best for doing things like
```php
TokensTable::whereIn('token', $tokens)->delete();
```
while other macro is best suited for per-token based logic (which i can't really think of, but decided to throw it in just in case).
If i were to choose, i'd rather have `devicesNotRegistered` over `deviceNotRegistered`, for optimisation reasons (with avorementioned example it would be 1 SQL query instead of, well, many) and you could still do `for` loop if you need to.